sources/osgShadow/ShadowReceiveAttribute.js

```diff
@@ -65,7 +69,7 @@ define( [
         attributeType: 'ShadowReceive',
 
         cloneType: function () {
-            return new ShadowReceiveAttribute( this._lightNumber, true );
+            return new ShadowReceiveAttribute( this._light, true );
```

Comment:
btw if lightnum change it will be detected by the check in State.js so maybe the light here does not add value for error checking